src/uwtools/tests/test_cli.py

```diff
@@ -480,6 +495,21 @@ def test__dispatch_fv3():
     )
 
 
+def test__dispatch_jedi():
+    cycle = dt.datetime.now()
+    args: dict = {
+        "batch": True,
+        "config_file": "config.yaml",
+        "cycle": cycle,
+        "dry_run": False,
+    }
+    with patch.object(uwtools.api.jedi, "execute") as execute:
+        cli._dispatch_jedi({**args, "action": "foo"})
+    execute.assert_called_once_with(
+        task="foo", config_file="config.yaml", cycle=cycle, batch=True, dry_run=False
```

Comment:
I'm trying to standardize the way `config` vs. `config_file` is used for the jedi driver to match other drivers. This was also the expected output vs. the actual output. If anyone has any suggestions to get this to conform better let me know. It wanted `config_file` here and not simply `config`.

My actual output the tests didn't like:
```
batch=True, config="config.yaml", cycle=cycle, dry_run=False, graph_file=None, task="foo"
```